MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -8,7 +8,9 @@ protocol FeedViewControllerDelegate {
 	func didRequestFeedRefresh()
 }
 
-public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView, FeedErrorView {
+	@IBOutlet public var errorView: ErrorView!
```

Comment:
While having `errorView` getter public it is still important to have the setter `private` to prevent clients from changing the instance. Please make the setter private while leaving the getter public

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView!
```

Reply:
Done